cmd/destroy/destroy.go

```diff
@@ -121,7 +121,12 @@ func Destroy(ctx context.Context) *cobra.Command {
 				options.ManifestPath = uptManifestPath
 			}
 			if err := contextCMD.LoadContextFromPath(ctx, options.Namespace, options.K8sContext, options.ManifestPath); err != nil {
-				return err
+				if err.Error() == fmt.Errorf(oktetoErrors.ErrNotLogged, okteto.CloudURL).Error() {
+					return err
+				}
+				if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{Namespace: options.Namespace}); err != nil {
+					return err
+				}
 			}
```

Comment:
Just for the record and other possible reviewers. This logic is already in the `deploy` command. That is the reason why the `deploy` is working but the `destroy` doesn't. This will probably need to be reviewed carefully and refactored, but enough to fix the destroy